content/ja/docs/reference/glossary/helm-chart.md

```diff
@@ -0,0 +1,19 @@
+---
+title: Helm Chart
```

Comment:
```suggestion
title: Helmチャート
```
I think it's better to use "チャート" because used in https://helm.sh/ja/docs/.
And, the document says that "chart" is not a proper noun.
[単語 Helm と Chart の使い方](https://helm.sh/ja/docs/chart_best_practices/conventions/#%E5%8D%98%E8%AA%9E-helm-%E3%81%A8-chart-%E3%81%AE%E4%BD%BF%E3%81%84%E6%96%B9)